en/concept/flight_modes.md

```diff
@@ -14,23 +14,23 @@ The sections below provide an overview of the modes, followed by a [flight mode
 "Manual" modes are those where the user has direct control over the vehicle via the RC control (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode. For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
 
 * **Fixed wing aircraft/ rovers / boats:** 
-  * **MANUAL:** The pilot's control inputs are passed directly to the output mixer.
-  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command.
+  * **MANUAL:** The pilot's control inputs or RC commands are passed directly to the output mixer.
+  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command. If the RC roll and pitch sticks are centered, the autopilot regulates the roll and pitch angles to zero, hence stabilizing (leveling-out) the attitude against any wind disturbances. However, in this mode the position of the aircraft is not controlled by the autopilot, hence the position can drift due to wind.
 * **Multirotors:**
-  * **ACRO:** The pilot's inputs are passed as roll, pitch, and yaw *rate* commands to the autopilot.  This allows the multirotor to become completely inverted.  Throttle is passed directly to the output mixer
-  * **RATTITUDE** The pilot's inputs are passed as roll, pitch, and yaw *rate* commands to the autopilot if they are greater than the mode's threshold.  If not the inputs are passed as roll and pitch  *angle* commands and a yaw *rate* command.  Throttle is passed directly to the output mixer.
-  * **STABILIZED** The pilot's inputs are passed as roll and pitch *angle* commands and a yaw *rate* command.  Throttle is passed directly to the output mixer.
+  * **ACRO:** The pilot's inputs are passed as roll, pitch, and yaw *rate* commands to the autopilot. The autopilot controls the angular rates, but not the attitude. Hence, if the RC sticks are centered the vehicle will not level-out. This allows the multirotor to become completely inverted.  Throttle is passed directly to the output mixer.
```

Comment:
So as I understand it, the autopilot passes *rate* commands for roll, pitch, yaw. Does this mean that the vehicle responds to speed of movement of the RC controls, not their position. If the RC is centred there will be no movement of sticks, hence no rate change,, so the vehicle would just keep its previous roll/pitch/yaw?

Personally I didn't find this new sentence helpful "The autopilot controls the angular rates, but not the attitude. Hence, if the RC sticks are centered the vehicle will not level-out. ". I think what you are trying to do is compare this to the "STABILIZED" mode? Essentially saying in stabilised mode the system will level you out, but in ACRO you're on your own. Is that correct?

Reply:
I think the "position" of RC sticks determines roll, pitch and yaw rate references to be sent to the autopilot "rate controller". This is what the PX4 team confirmed. So if the RC sticks are centered, it is passed as "zero" roll, pitch, yaw _rate_ reference to the autopilot. Which means we are telling the autopilot to not change the roll, pitch and yaw angles anymore, consequently (depending on the speed of the system response) trying to keep the current roll, pitch, yaw angles as they are. Whereas in Stabilised mode, centering the sticks position means we are sending zero roll,pitch, yaw _angle_ references so the controller tries to keep the angles zero.

Also, in the Acro mode, I personally think it is important to make this distinction between the stabilised and acro mode because one uses "attitude controller, meaning roll/pitch angles controller" vs the other which triggers "attitude _rate_ controller, meaning roll/pitch rate controller".

Reply:
Thank you for that description. So what I think you're saying is that with the yaw rate commands (say) we're controlling the rate of angular rotation of yaw while with the angle commands we're setting the target yaw angle. Angle commands are therefore better for stabilisation because the vehicle tracks the position of the sticks, while rate commands affect rate of change, and are hence more responsive. Correct?

Reply:
One correction in my comment above:
In Stabilised mode,  the "position" of RC sticks are translated into roll, pitch angles and yaw **rate** references to be sent to the autopilot's "attitude controller" and "yaw rate controller". Centering the sticks position means we are sending zero roll and pitch angles and zero yaw rate references so the controller tries to keep the roll, pitch angles and yaw rate zero.

Reply:
Awesome. OK, so please make sure that the stabilised doc reflects how it works :-)

Re this ACRO .. do you think this is an improvement:
* **ACRO:** The pilot's inputs control the roll, pitch, and yaw *angular rotation rate*. Throttle is passed directly to the output mixer. 
  The multirotor can become completely inverted. If the RC sticks are centered the vehicle will try to keep the angular rates at zero (maintain the vehicle's current attitude).